### PR TITLE
Added FTP to the factory

### PR DESCRIPTION
**Pull request by dgreda**

### CODE REVIEW

**Comment by dgreda** [comment]
Hey @titosemi please check

File: src/Config/Adapter/Ftp.php [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                    ->cannotBeEmpty()
+                ->end()
+                ->scalarNode(self::INDEX_USERNAME)
+                    ->isRequired()
+                    ->cannotBeEmpty()
+                ->end()
+                ->scalarNode(self::INDEX_PASSWORD)
+                    ->isRequired()
+                ->end()
+                ->scalarNode(self::INDEX_PORT)
```
Comment (titosemi) [inline src/Config/Adapter/Ftp.php]: Is the default port (21) handled by ```\League\Flysystem\Adapter\Ftp``` ?

**Comment by Umkus** [comment]
Could we _please_ try keeping ourselves in sync? Or at least not adding new code during weekends without a ticket, at least?
OMSI-846
https://github.com/Westwing-Home-and-Living/flysystem-factory/pulls

File: src/Config/Adapter/Ftp.php (hunk 1 above)
Comment (dgreda) [inline src/Config/Adapter/Ftp.php]: The port is optional parameter and is set properly by Flysystem driver to 21. Tested :)

Comment (titosemi) [inline src/Config/Adapter/Ftp.php]: I know that it's optional, just wanted to check that it's set as 21 if not specified by the FTP driver :)

**Comment by dgreda** [comment]
Code was reviewed and approved. Merged.

**Pull request merged**